app/routes/book_routes.py

```diff
@@ -0,0 +1,17 @@
+from flask import Blueprint
+from app.models.book import books
```

Comment:
I could see there being some questions about why don't we need to import Book in the routes file, but I think that will be a good reminder after Swap Meet where folks try to import more than they strictly "need." We can use an instance of a type without importing the name of the type.

Reply:
We remove the `books` variable import from `book_routes` when we remove the hardcoded content. We could choose to explicitly change the import in routes to `Book` on this branch instead of removing it since we'll be using it in the next branch. 

We would be importing it to `book_routes` without using it at this point in time, does that feel okay to avoid listing out dependencies that we don't strictly need to in `__init__`? Alternatively we could import `Book` in the `__init__` now, then move the import to `book_routes` on the next branch, but that feels more confusing to follow to me.

Reply:
Sorry, I wasn't suggesting a change, just making an observation. I agree with yout that I don't think we should import Book here. The right time to switch it over is when the routes file does make use of the name Book when building queries. That doesn't change the fact that we _will_ be using instances of Book (through the books list), which some folks may find confusing since we didn't need to import the name Book.

Reply:
I realized I forgot we were still looking at branch 1b and started talking about changes we could make at the 3a mark in the second paragraph. 

When looking forward at 3a where we introduce the `Book` class and remove the hardcoded data, are you thinking that it would be worth immediately adding the `Book` import to `book_routes` file so that we don't need to import the book file in `__init__`?

Reply:
Oh, that's right. First everything in the routes file gets commented out and the import line gets removed, doesn't it. And then it gets added back in in the section immediately following...

In that case, doing the import in `__init__.py` seems fine. There's no path to the import at that point, and more often than not, we do introduce a new model before writing its routes. Personally, once I do have an import path, I like to remove the models from the `__init__.py`, but it's probably best if the learn material just leaves the import there. We could always add a callout for folks to decide for themselves whether they prefer importing all their models there or not.

Reply:
That sounds good to me, and I like the idea of including that callout in the text for folks to decide how they want to handle it in their projects and keep it consistent.